README.md

```diff
@@ -1,26 +1,26 @@
-# 👋 Welcome to skore
+# 👋 Welcome to `skore`
```

Comment:
```suggestion
# 👋 Welcome to skore
```

Reply:
For the big title, just once, Cam says that `code` is fine

Reply:
ok!